dev-python/PythonQt/PythonQt-3.1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 )
```

Comment:
yes please do, py3.5 is `use.stable.mask`ed anyway, so stabilizing is not an argument not to add py3.5 support